code/modules/recycling/conveyor.dm

```diff
@@ -99,99 +125,135 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 		if(SOUTHWEST)
 			forwards = WEST
 			backwards = NORTH
-	if(verted == -1)
+
+	if(inverted)
+		var/temp = forwards
+		forwards = backwards
+		backwards = temp
+	// We need to do this this way to ensure good functionality on corner belts.
+	// Basically, this allows the conveyor belts that used a flipped belt sprite to
+	// still convey items in the direction of their arrows. It's different from inverted,
+	// so they need to be ran separately.
+	if(flipped)
 		var/temp = forwards
 		forwards = backwards
 		backwards = temp
-	if(operating == 1)
+	if(operating == CONVEYOR_FORWARD)
 		movedir = forwards
 	else
 		movedir = backwards
 	update()
 
 /obj/machinery/conveyor/update_icon_state()
-	icon_state = "[base_icon_state][(machine_stat & BROKEN) ? "-broken" : (operating * verted)]"
+	icon_state = "[base_icon_state][inverted ? -operating : operating ][flipped ? "-flipped" : ""]"
 	return ..()
 
+/**
+ * Proc to update the conveyor depending on if it's without power (there used to be more
+ * checks, but they were broken and thus removed).
+ *
+ * Returns TRUE if it is still able to be operating after the update, FALSE if not.
+ */
 /obj/machinery/conveyor/proc/update()
-	if(machine_stat & BROKEN || !operable || machine_stat & NOPOWER)
+	. = TRUE
+	if(machine_stat & NOPOWER)
 		operating = FALSE
-		update_appearance()
-		return FALSE
-	return TRUE
+		. = FALSE
+	update_appearance()
 
 // machine process
 // move items to the target location
 /obj/machinery/conveyor/process()
-	if(machine_stat & (BROKEN | NOPOWER))
+	if(machine_stat & NOPOWER)
 		return
 
-	//If the conveyor is broken or already moving items
+	// If the conveyor is off or already moving items.
 	if(!operating || conveying)
 		return
 
 	use_power(6)
 
-	//get the first 30 items in contents
+	// Get the first 30 items in contents.
 	var/turf/locturf = loc
 	var/list/items = locturf.contents - src
-	if(!LAZYLEN(items))//Dont do anything at all if theres nothing there but the conveyor
+	if(!LAZYLEN(items)) // Don't do anything at all if theres nothing there but the conveyor.
 		return
+
 	var/list/affecting
 	if(length(items) > MAX_CONVEYOR_ITEMS_MOVE)
-		affecting = items.Copy(1, MAX_CONVEYOR_ITEMS_MOVE + 1)//Lists start at 1 lol
+		affecting = items.Copy(1, MAX_CONVEYOR_ITEMS_MOVE + 1) // Lists start at 1 lol.
 	else
 		affecting = items
+
 	conveying = TRUE
 
-	addtimer(CALLBACK(src, .proc/convey, affecting), 1)//Movement effect
+	addtimer(CALLBACK(src, .proc/convey, affecting), 1) // Movement effect.
```

Comment:
Comments don't nessesarially need ., don't feel like you need to add em everywhere.
The space after is also just a style choice, but since you're redoing the whole file s ok

Reply:
> Comments don't nessesarially need ., don't feel like you need to add em everywhere. The space after is also just a style choice, but since you're redoing the whole file s ok

Oh, yeah, I know. I just thought I'd go ahead and do that, it's just personal preference at this point, I'm not too worried about it either way. Just space is just better for visibility in my opinion. The period, yeah, that doesn't honestly matter too much.